### DISCUSSION TITLE
about "q;"

### DISCUSSION THREAD

**Discussion by Kectai** [dbeaver/dbeaver#19295] [DBeaver Development]
<img width="267" alt="image" src="https://user-images.githubusercontent.com/46049250/223694391-991bcd05-76ce-4c90-8711-3544e15edf4e.png">

dbeaver community23.0.0

oracle19c

The old version used before did not have this problem.

**Comment by LonwoLonwo** [comment]
Yeah, strange story, thanks for the report.

As a workaround - you can add something on the end of the string like `WHERE 1=1`

![2023-03-10 13_07_23-DBeaver Ultimate 23 0 0 - _orcl_ Script-59](https://user-images.githubusercontent.com/45152336/224289268-5ba01f86-a656-4ed2-86cc-5f4374d3e6a0.png)

**Reply by Kectai** [reply]
ok, thanks
